manim/mobject/graph.py

```diff
@@ -98,65 +98,94 @@ def _determine_graph_layout(
 
 
 def _tree_layout(
-    G: nx.classes.graph.Graph,
+    T: nx.classes.graph.Graph,
     root_vertex: Union[Hashable, None],
-    scale: float,
-) -> dict:
-    result = {root_vertex: np.array([0, 0, 0])}
+    scale: Union[float, tuple] = 2,
+    orientation: str = "down",
```

Comment:
Is there any way to influence this parameter? Maybe borrowing sagemath's docs for this param + adding a test / a way to access it when creating the `Graph` would be nice.